domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/VurderAvslagGrunnetBeregning.kt

```diff
@@ -37,3 +25,11 @@ sealed class AvslagGrunnetBeregning {
         SU_UNDER_MINSTEGRENSE
     }
 }
+
+fun Merknad.Beregning.tilAvslagsgrunn(): AvslagGrunnetBeregning.Grunn {
+    return when (this) {
+        is Merknad.Beregning.BeløpErNull -> AvslagGrunnetBeregning.Grunn.FOR_HØY_INNTEKT
```

Comment:
Kunne man delt opp Merknader i 2 kategorier, hvor den ene var de som fører til avslag?

Reply:
Det er en mulighet å gjøre det slik også, absolutt. Et case som kanskje taler i mot det er om det ikke er "svart/hvitt" - altså at vi har contextuelle tilfeller hvor en merknad av "typen" avslag ikke skal gi avslag. Kanskje vi evt kunne vært litt mindre streng hvis dette er tilfellet? "KanGiAvslag"? Åpen for å jobbe med typing og navngiving her.

Reply:
🤷